openstf/tasks/utils/predictionjobloop.py

```diff
@@ -168,6 +171,7 @@ def map(self, function, *args, **kwargs):
                 "num_jobs": num_jobs,
                 "pids_successful": pids_successful,
                 "pids_unsuccessful": pids_unsuccessful,
+                "exceptions": pids_unsuccessful_dict,
```

Comment:
This can be remove/simplified if we use  a single data structure.

Reply:
yes